ygnmi/gnmi.go

```diff
@@ -270,7 +270,10 @@ func receiveStream[T any](sub gpb.GNMI_SubscribeClient, query AnyQuery[T]) (<-ch
 		for {
 			recvData, sync, err = receive(sub, recvData, true)
 			if err != nil {
-				errCh <- fmt.Errorf("error receiving gNMI response: %w", err)
+				select {
```

Comment:
nit: comment explaining why we're doing this

Reply:
Done :-)